src/examples/starter-apps/translator.py

```diff
@@ -50,6 +51,20 @@ async def process_frame(self, frame: Frame) -> AsyncGenerator[Frame, None]:
             yield frame
 
 
+class TranslationSubtitles(FrameProcessor):
+    def __init__(self, language):
+        self._language = language
+
+    async def process_frame(self, frame: Frame) -> AsyncGenerator[Frame, None]:
+        if isinstance(frame, TextFrame):
+            app_message = {
```

Comment:
Maybe I'm missing something, but should a frame containing this message be yielded? And should the TextFrame be yielded in addition to this message? (maybe not since this is the last frameprocessor in the pipeline)

Also, now that I'm thinking about this... should this frameprocessor come before the TTS processor, and yield the app message ahead of the textframe? That way the subtitle will show before the audio starts playing, which seems better than after.